### DISCUSSION TITLE
List of Models in Azure Open AI is not matching with what I can see in SAP Debugger

### DISCUSSION THREAD

**Discussion by shreyashagrwl** [microsoft/aisdkforsapabap#34] [Q&A]
Hi Gopal,

I am at **Installing the SDK --> First Test Run - Azure Open AI** step. In the step it asks to compare the list of models present in Azure Open AI Studio with the one seen in the SAP debugger. In my case, the model list isn't matching, what could have gone wrong?
Image from Azure Open AI Studio - 
![image](https://github.com/microsoft/aisdkforsapabap/assets/146458948/daeb162c-47cf-41c5-a850-f6f52380a72e)

 Image from SAP Debugger - 
![image](https://github.com/microsoft/aisdkforsapabap/assets/146458948/51a37c9b-f788-4674-8701-22f7df55016c)

**Comment by gopalnair** [comment]
Hi @shreyashagrwl ,
  I have to admit, I do not know the exact reason for this. I have observed this as well.

AI SDK for SAP uses REST API to query the list of models from AI Engine endpoint. The list in the debugging session is received from the AI Endpoint, as the available models.

However, when going through the web interface, a subset of this list is what gets presented. I have tried to see if there are any specific parameters that are being filtered in the web interface - but have not figured it out yet.

If/When I do eventually figure it out, I will be sure to put a response here.

Best Regards,
Gopal Nair.

**Reply by shreyashagrwl** [reply]
Thanks Gopal for your quick response. I have just started in this space and am exploring.
This this was the last step of installation, I am done. However, I am still not sure how to test what I have installed with different use cases. Any detailed video tutorial will be much helpful

**Reply by gopalnair** [reply]
Hi @shreyashagrwl ,
  A very good place to start would be to review the demo programs. The ABAP demo programs are written with lots of comments, so that every step of each functionality is explained in detail.

You can see the name of the demo programs in the documentation website.

For example - for Models, the demo program is mentioned in [Models Documentation] (https://microsoft.github.io/aisdkforsapabap/docs/working-with-sdk/models) 


I do plan on creating detailed video tutorials for each of these - as well as advanced functionality in near future.

So - as a starting point, please see if the demo Model program is understandable. And then, we can go forward from there. 

Please let me know how it goes.

Best Regards,
Gopal Nair.
